udev/revpi_mac

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+DEVNAME=$1
+OFFSET=$2
+
+read -r BASE_MAC < /proc/device-tree/hat/custom_5
```

Comment:
Idea: Check if ` /proc/device-tree/hat/custom_5` exists? Otherwise the script will fail with an unhandled error